dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.9.1-slim-buster as base
+
```

Comment:
You might want to set some env vars in the Dockerfile, like `FLASK_APP=todo_app/app` in the base stage and `ENV FLASK_ENV=production` in the production stage.